lib/src/fnc/args.rs

```diff
@@ -63,37 +63,37 @@ impl FromArg for Bytes {
 
 impl FromArg for i64 {
 	fn from_arg(arg: Value) -> Result<Self, Error> {
-		arg.convert_to_i64()
+		arg.coerce_to_i64()
 	}
 }
 
 impl FromArg for u64 {
 	fn from_arg(arg: Value) -> Result<Self, Error> {
-		arg.convert_to_u64()
+		arg.coerce_to_u64()
 	}
 }
 
 impl FromArg for f64 {
 	fn from_arg(arg: Value) -> Result<Self, Error> {
-		arg.convert_to_f64()
+		arg.coerce_to_f64()
 	}
 }
 
 impl FromArg for isize {
 	fn from_arg(arg: Value) -> Result<Self, Error> {
-		Ok(arg.convert_to_i64()? as isize)
+		Ok(arg.coerce_to_i64()? as isize)
 	}
 }
 
 impl FromArg for usize {
 	fn from_arg(arg: Value) -> Result<Self, Error> {
-		Ok(arg.convert_to_u64()? as usize)
+		Ok(arg.coerce_to_u64()? as usize)
 	}
 }
 
 impl FromArg for Vec<Number> {
 	fn from_arg(arg: Value) -> Result<Self, Error> {
-		arg.convert_to_array()?.into_iter().map(Value::convert_to_number).collect()
+		arg.coerce_to_array_type(&Kind::Number)?.into_iter().map(Value::try_into).collect()
```

Comment:
Just flagging this for later review into whether it affects performance (e.g. by allocating more).